src/network/netdev/netdev-gperf.gperf

```diff
@@ -77,6 +78,19 @@ Tunnel.ISATAP,                     config_parse_tristate,                0,
 FooOverUDP.Protocol,               config_parse_uint8,                   0,                             offsetof(FouTunnel, fou_protocol)
 FooOverUDP.Encapsulation,          config_parse_fou_encap_type,          0,                             offsetof(FouTunnel, fou_encap_type)
 FooOverUDP.Port,                   config_parse_ip_port,                 0,                             offsetof(FouTunnel, port)
+L2TP.TunnelId,                     config_parse_unsigned,                0,                             offsetof(L2tpTunnel, tunnel_id)
+L2TP.PeerTunnelId,                 config_parse_unsigned,                0,                             offsetof(L2tpTunnel, peer_tunnel_id)
+L2TP.SessionId,                    config_parse_unsigned,                0,                             offsetof(L2tpTunnel, session_id)
+L2TP.PeerSessionId,                config_parse_unsigned,                0,                             offsetof(L2tpTunnel, peer_session_id)
+L2TP.UDPSourcePort,                config_parse_l2tp_tunnel_port,        0,                             0
+L2TP.UDPDestinationPort,           config_parse_l2tp_tunnel_port,        0,                             0
```

Comment:
`offsetof(L2tpTunnel, l2tp_udp_sport)` and `offsetof(L2tpTunnel, l2tp_udp_dport)`.